### DISCUSSION TITLE
DraftVersionVisibility enum

### DISCUSSION THREAD

**Discussion by dabaker6** [pnp/pnpcore#1284] [Ideas] [answered]
I'm in the middle of migrating from an old tool using SharePointPnPCoreOnline. When enabling content approval in lists there was the following enum for who is able to see draft versions of items.

```c#
public enum DraftVisibilityType
{
  Reader, 
  Author,
  Approver
}
```

I can see the property on the list object in PnP Core SDK is an int, is it possible to have this as an enum?

https://pnp.github.io/pnpcore/api/PnP.Core.Model.SharePoint.IList.html#PnP_Core_Model_SharePoint_IList_DraftVersionVisibility

**Comment by jansenbe** [comment, marked as answer]
@dabaker6 : that indeed makes sense, created an issue to track this ask

**Reply by dabaker6** [reply]
Excellent thanks!
